google/cloud/spanner_dbapi/cursor.py

```diff
@@ -208,10 +208,7 @@ def __enter__(self):
         return self
 
     def __exit__(self, etype, value, traceback):
-        self.__clear()
-
-    def __clear(self):
-        self._connection = None
```

Comment:
We don't actually need to drop connection object, checking `is_closed` will do the job